LibraBFT/Yasm/Base.agda

```diff
@@ -45,6 +45,9 @@ module LibraBFT.Yasm.Base
     -- Finally, messages must carry an epoch id and might have an author
     part-epoch  : Part → EpochId
 
+    -- A decidable over PeerId's
+    _≟Peer_ : ∀ (p₁ p₂ : PeerId) → Dec (p₁ ≡ p₂)
```

Comment:
Funnily enough, just yesterday I did the same in the branch I am working on.  Could you move this to be the second parameter (right after `PeerId`), which will avoid a conflict with my similar change.  Also, please spell it like this:
```
  _≟PeerId_ : (p1 p2 : PeerId) → Dec (p1 ≡ p2)
```

Reply:
Done!